sdk/core/azure-core/inc/azure/core/azure_assert.hpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+/**
+ * @file
+ * @brief Deprecated assert macros. It is only temporarily made available for compatibility and
+ * should NOT be used by any callers outside of the SDK.
+ *
+ */
+
```

Comment:
Since we brought these back, updated the changelog entry:
https://github.com/Azure/azure-sdk-for-cpp/pull/3255